app/controllers/idv_controller.rb

```diff
@@ -32,6 +32,10 @@ def verify_identity
     redirect_to idv_welcome_url
   end
 
+  def handle_pending_profile
+    return redirect_to url_for_pending_profile_reason if user_has_pending_profile?
```

Comment:
I don't think the return is needed here.
```suggestion
    redirect_to url_for_pending_profile_reason if user_has_pending_profile?
```